packages/commonwealth/server/farcaster/frames/contest/contestPrizes.tsx

```diff
@@ -85,11 +85,14 @@ export const contestPrizes = frames(async (ctx) => {
     contestManager.interval === 0,
   );
 
+  // 10% fee deducted from prize pool
+  const netContestBalance = (Number(contestBalance) || 0) * 0.9;
```

Comment:
can you create some util that will be reused in those 3 places so that it will be easier to track and change in the future